jsk_rosbag_tools/python/jsk_rosbag_tools/cv_bridge_compat.py

```diff
@@ -0,0 +1,35 @@
+import importlib
+import os
+import sys
+
+
+# cv_bridge_python3 import
+if os.environ['ROS_PYTHON_VERSION'] == '3':
```

Comment:
why
```
import sys, os; sys.path.insert(0,'/opt/ros/' +  os.environ['ROS_DISTRO'] + '/lib/python3/dist-packages/')
import cv_bridge
from cv_bridge.boost.cv_bridge_boost import getCvType
```
did not work? 
c.f. https://github.com/ros/rosdistro/pull/32505#issuecomment-1075071759

Reply:
I no longer need this import because I no longer use catkin virtualenv.